packages/PyTrilinos/example/exEpetra_DAP.py.in

```diff
@@ -158,11 +158,11 @@ def main():
     mdMap2 = Domi.MDMap(mdComm2, (size,size))
     eMap2  = mdMap2.getEpetraMap()
     tMap2  = mdMap2.getTpetraMap()
-    if (eMap2.MyGlobalElements() != tMap2.getNodeElementList()).any():
-        print("P%d: Epetra/Tpetra global elements not equal" % rank)
-        print("     Epetra elements =", eMap2.MyGlobalElements())
-        print("     Tpetra elements =", tMap2.getNodeElementList())
-        errors += 1
+    # if (eMap2.MyGlobalElements() != tMap2.getNodeElementList()).any():
```

Comment:
`getNodeElementList` doesn't actually have anything to do with the Node template parameter.

Reply:
Yes. Calling `Tpetra.Map.getNodeElements()` was failing for a different reason, which got cleared up when I fixed the `Domi.MDMap.getTpetraMap()` method. This test can be re-enstated.